sdk/resourcemanager/monitor/armmonitor/models.go

```diff
@@ -791,43 +760,6 @@ type AzureMonitorPrivateLinkScopeProperties struct {
 	ProvisioningState *string `json:"provisioningState,omitempty" azure:"ro"`
 }
 
-// AzureResource - An azure resource object
-type AzureResource struct {
```

Comment:
What operation does this model belong to? Don't think it's associated with Metrics Client

Reply:
You could ignore this removal. It is an unreferenced model.